api/views.py

```diff
@@ -597,6 +597,8 @@ def set_subject(task):
         "task_link": reverse("teacher_task", kwargs=dict(task_id=task.pk)),
         "plagcheck_link": reverse("teacher_task_plagiarism", kwargs=dict(task_id=task.pk)),
         "plagiarism_key": task.plagiarism_key,
+        "type": task.type,
+        "task_types": [(item[0], item[1]) for item in Task.TaskType.choices],
```

Comment:
Let's just hardcode the list in the frontend, it's a bit weird to send a list of possible types with the task data. The frontend should also decide how to render the names of the task types, independently on the label of the DB 
value. (even if the string will be the same for now)